suricata.yaml.in

```diff
@@ -892,9 +892,10 @@ app-layer:
       detection-ports:
         dp: 443
 
-      # Generate JA3 fingerprint from client hello. If not specified it
+      # Generate JA3/JA4 fingerprints from client hello. If not specified it
       # will be disabled by default, but enabled if rules require it.
       #ja3-fingerprints: auto
+      #ja4-fingerprints: auto
```

Comment:
For quic, it is always enabled, right ?

Reply:
Yes. That's why it is only configured in the `tls` section. I guess nothing would stop us from making it configurable for QUIC too. I just matched what was done for JA3 there :)